docs/serving/revision-gc.md

```diff
@@ -0,0 +1,15 @@
+# Revision garbage collection
+
+Knative automatically cleans up inactive revisions as configured by the Operator. For more information, see the [Operator settings](../admin/serving/revision-gc.md).
+
+You can always retain a revision by adding the `serving.knative.dev/no-gc: "true"` annotation
```

Comment:
```suggestion
You can configure a revision so that it is never garbage collected by adding the `serving.knative.dev/no-gc: "true"` annotation:
```
I'd reword this to make it clearer - the always retain bit just reads weird to me. Might be just me though.